src/Page.js

```diff
@@ -530,11 +517,38 @@ Page.prototype.removeFrontMatter = function (includedPage) {
   return $.html();
 };
 
+/**
+  * Inserts the page layout's header to the start of the page
+  * @param pageData a page with its front matter collected
+  */
+Page.prototype.insertHeaderFile = function (pageData) {
+  const { header } = this.frontMatter;
+  let headerFile;
+  if (header) {
```

Comment:
Just wondering - could `header` be an empty string`''` (this would resolve to `false`)? If so, should do a check for `header === undefined` here instead to be more explicit?

Reply:
Yes, it could be an empty string but the user would have to **explicitly** specify a `header` field in `<frontmatter>` and leave it empty. 

Also, empty string is a [falsy value](https://developer.mozilla.org/en-US/docs/Glossary/Falsy), so it will still go to the `else` statement. 

Also the first statement is to check for any value specified within `<frontmatter>`, the logic of `header === undefined` is not correct (since it will be false when a valid string is given).

Reply:
Sorry, I think it should have been `header !== undefined` instead, but if the way we handle an empty header is the same as the way we handle the case where no header is defined, it might not be necessary.

Thanks for the clarification! 🙂